CHANGELOG.md

```diff
@@ -12,7 +12,7 @@
 
 ## 1.0.9
 
-* pyenv: Migrate project site from https://github.com/yyuu/pyenv to https://github.com/pyenv/pyenv
+* pyenv: Migrate project site from https://github.com/pyenv/pyenv to https://github.com/pyenv/pyenv
```

Comment:
Please fix a mistake in `CHANGELOG.md` when you get a chance. Then I will merge this. Thanks for contribution~